src/shout_subgroup/database.py

```diff
@@ -33,13 +33,12 @@ def configure_database() -> bool:
         logger.exception(f"Unable to connect to postgreSQL. See exception details ... {ex}")
         return False
 
-    global session
+    global Session
 
-    session_maker = sessionmaker(bind=engine)
-    session = session_maker()
+    Session = sessionmaker(bind=engine, expire_on_commit=False)
```

Comment:
`sessionmaker` creates a factory which we use to start db sessions. But the session isn't started until the function is called.

> a session establishes all conversations with the database and represents a “holding zone” for all the objects which you’ve loaded or associated with it during its lifespan

The mistake we made before was that we started a session, but we never closed it (manually or automatically).